src/app/printer/pdf/pdf.printer.ts

```diff
@@ -59,10 +60,24 @@ export class PdfPrinter implements Printer {
             const usageSheetWidthOffset = (width - refWidth - usageWidth) / 2;
             const usageSheetHeightOffset = (height - entries.length * usageLineHeight) / 2;
             Array.from(entries).forEach(([k, v], idx) => {
+                let paletteEntry = getPaletteEntryByColorRef(project.paletteConfiguration.palettes, k);
+
                 doc.rect(usageSheetWidthOffset, usageLineHeight * idx + usageSheetHeightOffset, refWidth, usageLineHeight);
                 doc.rect(usageSheetWidthOffset + refWidth, usageLineHeight * idx + usageSheetHeightOffset, usageWidth, usageLineHeight);
+                doc.rect(usageSheetWidthOffset + refWidth + usageWidth, usageLineHeight * idx + usageSheetHeightOffset, usageWidth, usageLineHeight);
+                
+                doc.setFillColor(paletteEntry.color.r,paletteEntry.color.g,paletteEntry.color.b);
+                doc.rect(usageSheetWidthOffset + refWidth + usageWidth + usageWidth / 4, usageLineHeight * idx + usageSheetHeightOffset + 1, usageWidth / 2, usageLineHeight -2, 'F');
```

Comment:
Curiosity, what does 'F' means as last parameter ? :thinking:

Reply:
Fill in the rectangle with the fill color. 😊

Reply:
Alright, I guess the PDF printer is getting old, I have absolutely no memories of that :sweat_smile: